### DISCUSSION TITLE
Feature request - Could we get some sort of indicator of whether the BIOS/CPU is x32/x64?

### DISCUSSION THREAD

**Discussion by Luk164** [ventoy/Ventoy#1210] [Ideas]
Recently I worked an a tablet that had a very peculiar design. The processor was an x64 Intel Atom, but the BIOS was x32. This caused me some trouble when I was trying to install Windows, as my image was x64. Could we get some sort of indicator to see whether we can load x64 images into the UI?

**Comment by ventoy** [comment]
When you in Ventoy's main menu. It will show in the bottom left of the screen:
```
www.ventoy.net BIOS 1.0.58    <------ This indicate the current BIOS is Legacy BIOS
www.ventoy.net UEFI 1.0.58    <------ This indicate the current BIOS is X86_64 UEFI
www.ventoy.net IA32 1.0.58    <------ This indicate the current BIOS is IA32 UEFI
```

For your tablet it should be the 3rd case.

**Reply by Luk164** [reply]
Thx, did not know about the IA32 thing. The CPU would still be nice addition though, even if it is rare for them to not match (*cough* ASUS *cough*)
